go/teams/service_helper.go

```diff
@@ -139,7 +139,7 @@ func userVersionsToDetails(ctx context.Context, g *libkb.GlobalContext, uvs []ke
 	for i, uv := range uvs {
 		pkg := packages[i]
 		active := true
-		if pkg.FullName != nil && pkg.FullName.EldestSeqno > uv.EldestSeqno {
+		if pkg.FullName != nil && pkg.FullName.EldestSeqno != uv.EldestSeqno {
```

Comment:
i think if `pkg.FullName.EldestSeqno == 0`, we want active to be `false`, no?  Should never have a reset user in the team.

Reply:
I maintain that is how this works, but I can change it if you want. We don't go into the `team_members` table rewrite all memberships rows to 0 when someone resets.

Reply:
ok, cool